### PR TITLE
feat: changes to website main navigation and footer navigation

### LINKED ISSUE #1786: Make changes to website main navigation and footer navigation
Change the main navigation and footer as per the mockup below:
Desktop: https://www.figma.com/file/PzWf8oALasWzgOBl4WhAcC/Homepage%2C-New-Business-Buyers-Page?node-id=568%3A8478&t=zZNVROHkxG3zu4AI-1
Mobile: https://www.figma.com/file/MuSpCtCdU2ns4cFAsPfvsx/Text-Styles-%26-Components-(current)?node-id=3%3A192&t=LV317zVpfEsy35WQ-1

Remove from the website:
- developers, scientists, validators, mainnet, community

Remove instagram and facebook from "join the community" section

Main nav links:
- Corporate buying - https://app.regen.network/buyers/
- Project developers- https://app.regen.network/land-stewards/ (would be great to change the url here to project-developers and do a redirect from /land-stewards
- Registry: https://regennetwork.notion.site/Welcome-to-Regen-Registry-0d55aab2a2d64f27aee2a468df172990
- Explore credits - app.regen.network/projects

Footer links:
- Marketplace: app.regen.network
- Registry: https://regennetwork.notion.site/Welcome-to-Regen-Registry-0d55aab2a2d64f27aee2a468df172990
- Built on Regen: https://regennetwork.notion.site/Built-On-Regen-Network-c6266114116842e389258747454f9f07
- Regen Foundation: https://regen.foundation/ [new tab]
- Cerulean Ventures: https://cerulean.vc/ [new tab]
- Program Guide: https://library.regen.network/v/regen-registry-program-guide/
- Create Methodology: https://library.regen.network/v/regen-registry-program-guide/methodology-development/methodology-development-overview
- Methodology Review: https://library.regen.network/v/regen-registry-program-guide/methodology-development/methodology-review-process
- Methodology Library: https://library.regen.network/v/methodology-library/
- Guides: guides.regen.network
- Governance: https://forum.regen.network/overview
- Docs: https://docs.regen.network/
- Whitepaper: https://regen-network.gitlab.io/whitepaper/WhitePaper.pdf
- $Regen Token: https://www.regen.network/token/
- FAQ: .https://www.regen.network/faq/
- Media: https://www.regen.network/media/
- Press Kit: https://www.regen.network/press-kit/
- Team: /team
- Careers: https://regennetwork.notion.site/Careers-at-Regen-Network-fe7d9645a39843cfb7eaceb7171d95af
- Contact: /contact

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-web#1786

Upgraded @regen-network/web-components
Changes in @regen-network/web-components: https://github.com/regen-network/regen-web/compare/7b7a8da..react-17

Also see: https://github.com/regen-network/regen-web/pull/1797

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

https://deploy-preview-2--regen-website.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,3 +23,5 @@ jobs:
 
       - name: build-www
         run: yarn build
+        env:
+          NODE_OPTIONS: '--openssl-legacy-provider --max-old-space-size=10216'
```
Comment (blushi) [inline .github/workflows/main.yml:27]: see explanation in https://github.com/regen-network/regen-web/pull/1796

**Comment by blushi** [comment]
@erikalogie you can test this on https://deploy-preview-2--regen-website.netlify.app/

**Comment by erikalogie** [comment]
This looks good to me. I am going to send to Sarah to confirm that everything looks good to her.

One thing that would be nice is to hide the "Regen Marketplace has launched" banner since we have the new button, it seems redundant.

**Comment by blushi** [comment]
> This looks good to me. I am going to send to Sarah to confirm that everything looks good to her.
> 
> One thing that would be nice is to hide the "Regen Marketplace has launched" banner since we have the new button, it seems redundant.

done

**Comment by erikalogie** [comment]
@blushi Sarah says this looks good to her, so I think it is ready to go

**Review by flagrede — APPROVED** [review]
Looks good!
Should we backport changes made to react-17 web-components to the main branch ?
It might create regression otherwise when we'll switch back to the main branch.

**Comment by blushi** [comment]
> Looks good! Should we backport changes made to react-17 web-components to the main branch ? It might create regression otherwise when we'll switch back to the main branch.

good point, what about backporting to dev first and then those changes will be integrated into main in the next release?

**Comment by flagrede** [comment]
> > Looks good! Should we backport changes made to react-17 web-components to the main branch ? It might create regression otherwise when we'll switch back to the main branch.
> 
> good point, what about backporting to dev first and then those changes will be integrated into main in the next release?

Sounds good !

**Comment by blushi** [comment]
> > > Looks good! Should we backport changes made to react-17 web-components to the main branch ? It might create regression otherwise when we'll switch back to the main branch.
> > 
> > 
> > good point, what about backporting to dev first and then those changes will be integrated into main in the next release?
> 
> Sounds good !

cool will open a PR later today or tmr

**Pull request merged**